src/ts2fable.fs

```diff
@@ -119,12 +123,27 @@ if argv |> List.exists (fun s -> s = "splitter.config.js") then // run from buil
     printfn "done writing test-compile files"
 
 else
-    printfn "ts2fable %s" Version.version
-
-    let tsfiles = argv |> List.filter (fun s -> s.EndsWith ".ts")
-    let fsfile = argv |> List.tryFind (fun s -> s.EndsWith ".fs")
+    let argv =
+        yargs
+            .usage("Usage: ts2fable some.d.ts src/Some.fs")
+            .command(U2.Case1 "$0 [files..]", "")
```

Comment:
This says `files` is varags https://github.com/yargs/yargs/blob/master/docs/advanced.md#variadic-positional-arguments